sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/placement/MarathonConstraintParser.java

```diff
@@ -247,9 +248,9 @@ public PlacementRule run(StringMatcher taskFilter, String fieldName, String oper
                                  Optional<String> requiredParameter) throws IOException {
             String parameter = validateRequiredParameter(operatorName, requiredParameter);
             if (isHostname(fieldName)) {
```

Comment:
There's probably some kind of clean subtype thing to do here, but I don't know what it is off the top of my head.

Reply:
More PRs coming.  Don't worry.